doc/nrf/ug_nrf5340.rst

```diff
@@ -106,165 +85,246 @@ Zephyr includes the `OpenAMP`_ library, which provides a complete solution for e
 The IPC peripheral is presented to Zephyr as an Interprocessor Mailbox (IPM) device.
 The OpenAMP library uses the IPM SHIM layer, which in turn uses the IPC driver in `nrfx`_.
 
-.. |note| replace:: To upgrade the firmware on the network core, perform the steps for FOTA upgrade described below, replacing :file:`app_update.bin`, which is the file used when upgrading firmware on the application core, with :file:`net_core_app_update.bin`.
+.. |note| replace:: The following instructions are for the application core.
+   To upgrade the firmware on the network core, perform the steps for FOTA upgrade described below, replacing :file:`app_update.bin`, which is the file used when upgrading firmware on the application core, with :file:`net_core_app_update.bin`.
    In addition, ensure that :option:`CONFIG_PCD` is enabled for the MCUboot child image.
    For more details, see :ref:`nc_bootloader`.
 
+Protocols and use cases
+***********************
 
-.. include:: ug_nrf52.rst
-   :start-after: fota_upgrades_start
-   :end-before: fota_upgrades_end
+nRF5340 samples usually consist of two separate images: one that runs on the network core and one that runs on the application core.
+For specific use cases, it is also possible to use only one of the cores.
 
-Multiprotocol support
-*********************
+The following sections describe the recommended architecture for using different protocols on the nRF5340 and list the provided samples.
 
-nRF5340 supports running another protocol in parallel with the :ref:`nrfxlib:softdevice_controller`.
-See the :ref:`ug_multiprotocol_support` user guide for instructions on how to enable multiprotocol support for Thread or Zigbee in combination with Bluetooth.
-
-The :ref:`nrfxlib:mpsl` library provides services for multiprotocol applications.
+Bluetooth Low Energy
+====================
 
-Available samples
-*****************
+.. list-table::
+   :header-rows: 1
 
-nRF5340 samples usually consist of two separate images: one that runs on the network core and one that runs on the application core.
+   * - Network core
+     - Application core
+   * - :ref:`zephyr:bluetooth-hci-rpmsg-sample`
+     - | :ref:`Bluetooth Low Energy samples <ble_samples>`
+       | :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`
 
-Most samples that support nRF5340 are not dedicated exclusively to this device, but can also be built for other chips that have the radio peripheral (for example, the nRF52 Series).
-The following subsections give a general overview of what samples can run on nRF5340.
+When using Bluetooth Low Energy on the nRF5340, you should run the Bluetooth LE Controller on the network core and the host part of the Bluetooth LE stack and the application logic on the application core.
```

Comment:
Shall we mention the alternative architecture, where the whole bluetooth stack runs on the network core? E.g.
`Alternatively, you can run the whole bluetooth stack on the network core using :ref:'ble_rpc' (supported for development).`
We might add that not all bluetooth APIs are supported yet - but that will require changing in future.
This would require further changes in the same including the table below. Shall we make this visible now, or not mention here since it's experimental and not finalized (just partial Bluetooth Host API support)?

Reply:
Looks good.